LinkDotNet.ValidationExtensions.Tests/RangeAttributes/GenericDynamicRangeTests.cs

```diff
@@ -0,0 +1,186 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using FluentAssertions;
+using Xunit;
+
+namespace LinkDotNet.ValidationExtensions.Tests.Generic;
+
+public partial class GenericDynamicRangeTests
```

Comment:
I noticed now that all tests are `partial`. Is there any reason ?

Reply:
No, at the beginning I created one class for all `DynamicRangeAttribute<T>` tests. But they had to separate.

It's done [8cffb1f](https://github.com/linkdotnet/ValidationExtensions/pull/7/commits/8cffb1f096a97f9301d4bdc180b6de74f8177f8b)